Dockerfile.win10.min

```diff
@@ -95,12 +92,11 @@ ARG CUDA_INSTALL_ROOT_WP="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v${
 
 WORKDIR /
 
-ARG TENSORRT_VERSION=8.5.1.7
-ARG TENSORRT_ZIP="TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-11.8.cudnn8.6.zip"
```

Comment:
Is `TENSORRT_ZIP` deleted by accident? I'm seeing `TENSORRT_ZIP` being used below.

Reply:
Yes, it was removed by mistake